<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="STSLP95" ignored="0" name="BankingApp Test" tests="6" failures="1" timestamp="01 Aug 2019 12:39:46 GMT" time="44.019" errors="0">
  <testcase name="loginTest" time="3.799" classname="com.bugzilla.testCases.TC_LoginTest_001"/>
  <testcase name="loginDDT" time="6.039" classname="com.bugzilla.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="5.718" classname="com.bugzilla.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="5.752" classname="com.bugzilla.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="5.713" classname="com.bugzilla.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="4.679" classname="com.bugzilla.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.bugzilla.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:39)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- BankingApp Test -->
